apps/SageAccountWeb/src/pages/AuthenticatedLayout.tsx

```diff
@@ -0,0 +1,24 @@
+import Footer from '@/components/Footer'
+import { Outlet } from 'react-router'
+import { useSynapseContext } from 'synapse-react-client/utils/context/SynapseContext'
+import LoginPage from './LoginPage'
+
+function AuthenticatedLayout() {
```

Comment:
re-implementation of the logic that renders a login page on subroutes when not logged in